docs/pages/reference/join-methods.mdx

```diff
@@ -0,0 +1,427 @@
+---
+title: Join Methods and Token Reference
+description: Describes the different ways to configure a node to join the Teleport cluster.
+keywords: [join, joining, token]
+---
+
+<Admonition type="note" title="Prerequisite">
+You must be familiar with the [Teleport Core concepts](../core-concepts.mdx) before reading this page.
+</Admonition>
+
+## Definitions
+
+### Joining
+
+Joining a Teleport cluster is the act of establishing trust between a new node and
+all the existing nodes, already part of the Teleport cluster. This trust is
+established via certificates. Once a node has its certificates, it can interact
+with the other Teleport nodes.
+
+To request its certificates, a node must prove its identity to the Auth service.
+Teleport offers multiple ways for a joining node to prove its authenticity, they
+are called the Join Methods.
+
+### Join Methods
+
+A join method is a way for the Auth server to validate that a node requesting to
+join the Teleport cluster is legitimate. Some join methods are universal while
+others rely on the joining node context. For example cloud-provider
+join-methods (such as `iam`, `gcp` or `azure`) or CI-provider (such as `github`,
+`gitlab`, `cicleci`, `jenkins`) are more flexible and provide
```

Comment:
Do we have a `jenkins` join method?

Reply:
Nope, I was misled by the machine id guide. But it's only using a regular token.